producer/src/block/block.go

```diff
@@ -78,12 +91,40 @@ func getMerkleRoot(l *list.List) []byte {
 // Concatenate all the fields of the block **header**
 // Return the SHA-256 hash of that concatenation
 func HashBlock(b Block) []byte {
-	// TODO
-	return []byte{}
+	blength := 4 + 8 + 8 + 32 + 32 // calculate the total length of the slice
```

Comment:
Let's make this a constant to reduce operation cost

Reply:
Done